README.md

```diff
@@ -37,10 +37,16 @@ This language server operates on views with the `source.rust` base scope.
 
 ## Custom Command Palette Commands
 
+### LSP-rust-analyzer: Run
+
+Select a command from the submenu.
+
+![Example](https://i.imgur.com/6bNxcur.gif)
```

Comment:
Can you instead include the image in the repo?

Reply:
Oh right sure